src/app/pages/products/components/product-card/product-card.component.html

```diff
@@ -13,6 +17,12 @@ <h3 class="product-name">{{ card.name }}</h3>
 </div>
 
 <div class="button-wrapper">
+  <app-wishlist-button
+    [isProductOnWishlist]="card.isOnWishlist"
+    [productId]="card.id"
+    [isOnProductPage]="!isExpanded">
+  </app-wishlist-button>
```

Comment:
change the `isProductOnWishlist` field's name to another one, our component should be reusable and shouldn't depend on outer pages.